src/ydata/sdk/common/client/client.py

```diff
@@ -160,8 +160,10 @@ def get_static_file(
         Returns:
             Response object
         """
+        from urllib.parse import urlparse
```

Comment:
is there a reason why this could not be imported at the beginning of the file?

Reply:
no no, it's just not used all the time, so i was trying to improve the interpretation of the file by the python. 
but i'm ok by moving this to the top of the file